modules/virt-configuring-node-exporter-service.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events-monitoring/virt-exposing-custom-metrics-for-vms.adoc
+
+[id="virt-configuring-node-exporter-service_{context}"]
+= Configuring the node exporter service
+
+The `node-exporter` agent is deployed on every virtual machine in the cluster from which you want to collect metrics. Configure the `node-exporter` agent as a service to expose internal metrics and processes that are associated with virtual machines.
+
+.Prerequisites
+
+* Install the {product-title} CLI `oc`.
+* Log in to the cluster as a user with `cluster-admin` privileges.
+* Create the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` project.
+* Configure the `user-workload-monitoring-config` `ConfigMap` object in the `openshift-user-workload-monitoring` project by adding `enableUserWorkload` and setting its value to `true`.
+
+.Procedure
+
+. Create the `node-exporter` YAML file. In the following example, the file is called `node-exporter-service.yaml`.
++
+[source,yaml]
+----
+kind: Service
+apiVersion: v1
+metadata:
+  name: node-exporter-service <1>
+  namespace: dynamation <2>
+  labels:
+    servicetype: metrics <3>
+spec:
+  ports:
+    - name: exmet <4>
+      protocol: TCP
+      port: 9100 <5>
+      targetPort: 9100 <6>
```

Comment:
Add # before the no.s like `name: node-exporter-service #<1>` 
so it will not give any problem if someone copy and paste same content.

Reply:
@sbulage - What you see here is standard practice - please refer to OpenShift documentation. This is the standard approach for referencing an item in a YAML file by number. Cannot add a  # sign - that would be deviation from standard OpenShift doc practice. You are welcome to open a defect on OpenShift documentation if you want this practice of adding a 3 sign to be followed.